packages/bpk-component-code/src/BpkCodeBlock.js

```diff
@@ -32,21 +32,18 @@ type Props = {
 };
 const BpkCodeBlock = (props: Props) => {
   const { children, alternate, className, ...rest } = props;
-  const preClassNames = [getClassName('bpk-code__pre')];
-  const codeClassNames = ['bpk-code', 'bpk-code--block'].map(getClassName);
-
-  if (alternate) {
-    preClassNames.push(getClassName('bpk-code__pre--alternate'));
-  }
+  const preClassNames = getClassName(
+    'bpk-code__pre',
+    alternate && 'bpk-code__pre-alternate',
+    className,
+  );
 
-  if (className) {
-    preClassNames.push(className);
-  }
+  const codeClassNames = getClassName('bpk-code', 'bpk-code--block');
 
   return (
     // $FlowFixMe - inexact rest. See 'decisions/flowfixme.md'.
-    <pre className={preClassNames.join(' ')} {...rest}>
-      <code className={codeClassNames.join(' ')}>{children}</code>
+    <pre className={preClassNames} {...rest}>
+      <code className={codeClassNames}>{children}</code>
```

Comment:
I updated this to fix some Flow errors that came up when I added Flow to `bpk-react-utils`.